addons/info/src/components/PropTable.js

```diff
@@ -6,6 +6,9 @@ import { Table, Td, Th } from '@storybook/components';
 import PropVal from './PropVal';
 import PrettyPropType from './types/PrettyPropType';
 
+const willItOccurAgain = (collection, item, itemIndex = 0) =>
+  collection.indexOf(item, itemIndex + 1) !== -1;
```

Comment:
Also a bit hesitant to use this to generate a key, seems a bit unperformant and a bit of an overkill

Reply:
I think that would be good! It'll simplify the logic which is good